src/schema/purchaseOrder/actions.tsx

```diff
@@ -337,10 +381,6 @@ export const createPaymentIntent = async ({
   currencyId: string;
   logger: Logger;
 }) => {
-  if (!USER) {
```

Comment:
El USER está tipado como not null, no hace falta este check

Reply:
Estás seguro? Confío en ti!

Reply:
Sí, seguro. TypeScript garantiza que USER no puede ser null/undefined aquí.

Si en algún punto intentáramos llamar a esta función sin USER o con USER null/undefined, TypeScript nos daría un error de compilación.

Mantener este check en runtime creo que sería redundante y va contra el principio de confianza en el system de tipos.